_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -32,11 +32,11 @@ This rule applies to any element with an `aria-hidden` [attribute value][] of `t
 
 ## Expectation
 
-None of the target elements are [focusable][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are [focusable][].
+None of the target elements are part of the [sequential focus navigation][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are part of the [sequential focus navigation][], unless, upon receiving focus, focus is moved to an element that does not fail this rule.
```

Comment:
I'm not sure about how the redirect exception is written currently.
```html
<div aria-hidden="true">
  <a href="…" onfocus='document.getElementById("next").focus()'>Redirect</a>
  <a href="…" id="next" tabindex="-1">Not tabbable?</a>
</div>
```
would pass the rule since the tabbable element redirect focus to an element that "does not fail the rule" (since it's not tabbable, hence ignored by the rule)

Even worse, the rule targets the `div` (the element with `aria-hidden`), not the links. So even if the second `a` was tabbable, it would "not fail the rule" since only the `div` can fail the rule.

What about "focus is moved to an element which is [included in the accessibility tree][]"?

Reply:
I agree. Will fix.